docker/.env.example

```diff
@@ -1,18 +1,28 @@
 # Secrets
 # YOU MUST CHANGE THESE BEFORE GOING INTO PRODUCTION
+# used as a secret to verify the JWT token signature
 JWT_SECRET=your-secret
-STORE_ENCRYPTION_KEY="<ENCRYPTION_KEY_MUST_BE_32_LONG>"
+# used to encrypt/decrypt the provider credentials
+STORE_ENCRYPTION_KEY=<ENCRYPTION_KEY_MUST_BE_32_LONG>
+
+# Host
+HOST_NAME=http://localhost
 
 # General
-NODE_ENV=local
-MONGO_URL=mongodb://mongodb:27017/novu-db
+NODE_ENV=production
 MONGO_MAX_POOL_SIZE=500
+MONGO_MIN_POOL_SIZE=100
+# MONGO USER
+MONGO_INITDB_ROOT_USERNAME=root
+# MONGO PASSWORD
+MONGO_INITDB_ROOT_PASSWORD=secret
+MONGO_URL=mongodb://root:secret@mongodb:27017/novu-db?authSource=admin
 REDIS_HOST=redis
 REDIS_PASSWORD=
 REDIS_CACHE_SERVICE_HOST=
 
 # AWS
-S3_LOCAL_STACK=http://localhost:4566
+S3_LOCAL_STACK=$HOST_NAME:4566
```

Comment:
i was not aware of this syntax in the.env file, looks good, love the reusability 😉